tests/TestCase/Error/DebuggerTest.php

```diff
@@ -245,66 +147,6 @@ public function testOutputAsException()
     }
 
     /**
-     * Tests that changes in output formats using Debugger::output() change the templates used.
-     *
-     * @return void
-     */
-    public function testAddFormat()
-    {
-        set_error_handler('Cake\Error\Debugger::showError');
-        $this->_restoreError = true;
-
-        Debugger::addFormat('js', [
-            'traceLine' => '{:reference} - <a href="txmt://open?url=file://{:file}' .
-                '&line={:line}">{:path}</a>, line {:line}'
-        ]);
-        Debugger::outputAs('js');
-
-        $result = Debugger::trace();
-        $this->assertRegExp('/' . preg_quote('txmt://open?url=file://', '/') . '(\/|[A-Z]:\\\\)' . '/', $result);
-
-        Debugger::addFormat('xml', [
-            'error' => '<error><code>{:code}</code><file>{:file}</file><line>{:line}</line>' .
-                '{:description}</error>',
-        ]);
-        Debugger::outputAs('xml');
-
-        ob_start();
-        $foo .= '';
-        $result = ob_get_clean();
-
-        $expected = [
-            '<error',
-            '<code', '8', '/code',
-            '<file', 'preg:/[^<]+/', '/file',
-            '<line', '' . ((int)__LINE__ - 7), '/line',
-            'preg:/Undefined variable:\s+foo/',
-            '/error'
-        ];
-        $this->assertHtml($expected, $result, true);
```

Comment:
No worries I spent some time on removing the deprecated code last night as well, but got too tire before I finished. However, I did get these tests working, so we can still use that part of it.